server/app/services/program/EligibilityDefinition.java

```diff
@@ -0,0 +1,40 @@
+package services.program;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.google.auto.value.AutoValue;
+import services.program.predicate.PredicateDefinition;
+
+/**
+ * Defines eligibility configuration on questions in a program.
+ *
+ * <p>Eligibility rules use the {@code PredicationDefinition} format, and indicate when an applicant
```

Comment:
```suggestion
 * <p>Eligibility rules use the {@link PredicateDefinition} format, and indicate when an applicant
```

Reply:
LOL, I kept making a different typo so this one was out of my focus